sanic/server.py

```diff
@@ -189,6 +193,14 @@ def write_response(self, response):
                 response.output(
                     self.request.version, keep_alive,
                     self.request_timeout))
+            if self.has_log:
```

Comment:
actually calling logging functions makes sanic a lot slower, even if it does nothing.
i think this is necessary for performance purpose.